src/routes/RootLayout.js

```diff
@@ -1,32 +1,19 @@
+import { FaUser } from 'react-icons/fa';
 import { NavLink, Outlet } from 'react-router-dom';
 
-const header = {
-  display: 'flex',
-  alignItems: 'center',
-  paddingTop: ' 1%',
-  paddingLeft: '5%',
-  paddingRight: '5%',
-  marginBottom: '4%',
-};
-const nav = {
-  padding: '0 10px',
-  fontSize: '1.2rem',
-  textDecoration: 'none',
-};
-const logo = {
-  color: '#0290FF',
-};
-
 function RootLayout() {
   return (
     <div>
-      <div style={header}>
-        <h1 style={logo}>Bookstore CMS</h1>
+      <header>
+        <div className="nav-container">
+          <h1 className="logo">Bookstore CMS</h1>
+          <NavLink className="nav books" to="/">BOOKS</NavLink>
+          <NavLink className="nav category" to="category">CATEGORIES</NavLink>
+        </div>
         <div>
-          <NavLink style={nav} to="/">Books</NavLink>
-          <NavLink style={nav} to="category">Categories</NavLink>
+          <FaUser className="user" size={14} />
```

Comment:
- [ ] As requested by the previous code reviewer, your design should match  Zeplin's design. When you check the navbar of your app, the alignement is not aligned to the design. Kindly fix it 👍 

### Your design

![Capture](https://github.com/MargaretTechworld/bookstore/assets/80458199/f65d5eda-a778-45ba-8ef7-ce26979ac718)

### Zeplin's design

![image](https://github.com/MargaretTechworld/bookstore/assets/80458199/7777b04a-d257-4974-9ce5-935a1d8126e1)